### DISCUSSION TITLE
40% keymap for Keyboardio Model 100

### DISCUSSION THREAD

**Discussion by giscus** [cdwilson/cdwilson.dev#9] [Articles]
# 40% keymap for Keyboardio Model 100

After backing the wildly successful Keyboardio Model 100 Kickstarter campaign last year, I finally received my keyboard in the mail 🥳
The keyboard is ✨beautiful✨ and the custom-sculpted keycaps feel really great to type on. After using the Model 100 for a couple weeks now, I’ve finally settled on a 40%-ish (3 rows of keys) keymap that feels really natural for me.
Although this keymap works really well for me personally, it’s probably not the right keymap for everyone.

https://cdwilson.dev/articles/40-percent-keymap-for-keyboardio-model-100/

**Comment by ken-olstad** [comment]
Thanks for the extensive write-up. A year or so later, how has this been working out?

**Reply by cdwilson** [reply]
@ken-olstad I'm still using this layout as my daily driver and still liking it!

I've made one very minor change to the layout: changing `~` to `Caps Lock` on layer 2 because `~` is already available on layer one as `shift`+`` ` ``.

There is one thing I don't like about this layout, but haven't bothered to fix yet: when I'm attempting to type a `1` (via `Layer #2 modifier`+`Q`), I find myself occasionally typing `⌘`+`Q` which quits the app I'm currently using... drives me CRAZY!
